src/main.zig

```diff
@@ -78,7 +78,7 @@ pub fn serialize(comptime T: type, allocator: Allocator, data: T, list: *ArrayLi
                     var length = tlist.items.len;
 
                     const length_length = try writeLengthLength(length, list);
-                    list.items[index] = 183 + length_length;
+                    list.items[index] = 247 + length_length;
```

Comment:
The other one makes sense but if this one is correctm then linke 74 should see a `192` and not `128`, since 128 + 55 == 183 and 192 + 55 == 247

Reply:
yeah, reading again, I'm convinced that it should be 192 on line 74.

Reply:
Agree. Fixed there too.